package.json

```diff
@@ -14,6 +14,8 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "@turf/line-intersect": "^6.5.0",
```

Comment:
Let's do an `npm uninstall @turf/line-intersect`. We already have the entire turf module in `@turf/turf`, so the line intersect is extra!